storage/memory/storage.go

```diff
@@ -178,6 +178,13 @@ func (o *ObjectStorage) ForEachObjectHash(fun func(plumbing.Hash) error) error {
 	return nil
 }
 
+func (o *ObjectStorage) ObjectPacks() ([]plumbing.Hash, error) {
+	return nil, nil
+}
+func (o *ObjectStorage) DeleteObjectPackAndIndex(plumbing.Hash) error {
+	return errNotSupported
```

Comment:
I think `return nil` is more inline with the other memory functions.  Not sure we need to cause an error here.